tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -522,6 +616,8 @@
 					<exclude name="tlc2/tool/UserModuleOverrideTest.java" />
 					<exclude name="tlc2/tool/UserModuleOverrideFromJarTest.java" />
 					<exclude name="tlc2/tool/UserModuleOverrideAnnotationTest.java" />
+					<!-- Bind to a port on the system so cannot be run in parallel. -->
```

Comment:
IIRC it is possible to configure the ports used RMI. Probably not worth it, though.

Reply:
That could be interesting for simplifying the ant build file! These tests collectively complete in under 10 seconds though.